packages/metrics-app/src/utils/address.ts

```diff
@@ -0,0 +1,19 @@
+import { getCurrentCoordinates } from '../services/devicelocation'
+
+
+const BIGDATACLOUD_API_KEY = 'bdc_ec1c6c097de9484c8db9633444fb8cca'
+
+export async function getAddress(): Promise<string | null> {
```

Comment:
Perhaps this could be called `getArea()` instead to make it clearer what it does.